charts/kueue/templates/webhook/manifests.yaml

```diff
@@ -75,6 +75,10 @@ webhooks:
         name: '{{ include "kueue.fullname" . }}-webhook-service'
         namespace: '{{ .Release.Namespace }}'
         path: /mutate-batch-v1-job
+    {{- if (hasKey $managerConfig "managedJobsNamespaceSelector") -}}
```

Comment:
Can you please fix it for all CRDs for consistency? Also, inject `namespaceSelector` consistently, for deployment it seems just above `rules`.

Reply:
Yes I will do it :+1:

Reply:
Adjusted, let me know.